hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/TriggersITest.groovy

```diff
@@ -87,6 +95,216 @@ class TriggersITest extends AbstractITestBase {
     }
 
     @Test
+    void testGroupTrigger() {
+        Trigger groupTrigger = new Trigger("group-trigger", "group-trigger");
+        groupTrigger.setEnabled(false);
+
+        // remove if it exists
+        def resp = client.delete(path: "triggers/groups/group-trigger", query: [keepNonOrphans:false,keepOrphans:false])
+        assert(200 == resp.status || 404 == resp.status)
+
+        // create the group
+        resp = client.post(path: "triggers/groups", body: groupTrigger)
+        assertEquals(200, resp.status)
+
+        resp = client.get(path: "triggers/group-trigger")
+        assertEquals(200, resp.status)
+        groupTrigger = (Trigger)resp.data;
+        assertEquals( true, groupTrigger.isGroup() );
+
+        ThresholdCondition cond1 = new ThresholdCondition("group-trigger", "DataId1-Token",
+            ThresholdCondition.Operator.GT, 10.0);
+        Map<String, Map<String, String>> dataIdMemberMap = new HashMap<>();
+        MemberCondition memberCondition = new MemberCondition( cond1, dataIdMemberMap );
+
+        resp = client.post(path: "triggers/groups/group-trigger/conditions", body: memberCondition)
+        assertEquals(resp.toString(), 200, resp.status)
+        assertEquals(1, resp.data.size())
+
+        // create member 1
+        Map<String,String> dataIdMap = new HashMap<>(1);
+        dataIdMap.put("DataId1-Token", "DataId1-Child1");
+        MemberTrigger memberTrigger = new MemberTrigger("group-trigger", "member1", "member1", null, dataIdMap);
```

Comment:
Same here, MemberTrigger sounds to me that is part of a group. But this class is used to map the members of the TriggerGroup, so I also wonders if TriggerMembers could sound more descriptive. I don't know.